src/Microsoft.AspNetCore.Blazor.Server/Circuits/Circuit.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Blazor.Server.Circuits
+{
+    /// <summary>
+    /// Represents an active connection between a Blazor server and a client.
+    /// </summary>
+    public class Circuit
+    {
+        internal Circuit(CircuitHost circuitHost)
+        {
+
+        }
```

Comment:
Note that there's nothing here yet on purpose. This class is for user-code - `CircuitHost` is for us.

Reply:
I'm curious about what sort of thing would eventually go here.

Reply:
Anything that user-code needs to be able to get from the circuit. So that would be things like persistance, or lifecyle notifications.